### ISSUE TITLE
Chrome crashes when using Lighthouse for specific sites 

### ISSUE THREAD

**Issue by tlansbergen-itrs** [GoogleChrome/lighthouse#16184]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).
- [X] Yes, my issue is not answered by [other FAQs](https://github.com/GoogleChrome/lighthouse#faq).

### URL

https://www.biggerbooks.com

### What happened?

During the Lighthouse run the page crashed with 'Aw, Snap! Error code: STATUS_ACCESS_VIOLATION'. 

Lighthouse reports: 
_There were issues affecting this run of Lighthouse:

There may be stored data affecting loading performance in this location: IndexedDB. Audit this page in an incognito window to prevent those resources from affecting your scores.
Browser tab has unexpectedly crashed._

### What did you expect?

The page not to crash and output instead of errors in Lighthouse

### What have you tried?

Switching mobile/desktop mode doesn't make a difference. Loading the page outside of Lighthouse is fine, as well as manually analysing the F12 Network tab / console etc. I can reproduce the same issue using the F12 Performance tab.

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

12.2.0

### Chrome Version

129

### OS

Windows 11

### Relevant log output

```sh
{
  "lighthouseVersion": "12.2.0",
  "requestedUrl": "https://www.biggerbooks.com/",
  "mainDocumentUrl": "https://www.biggerbooks.com/",
  "finalDisplayedUrl": "https://www.biggerbooks.com/",
  "finalUrl": "https://www.biggerbooks.com/",
  "fetchTime": "2024-09-12T09:30:15.076Z",
  "gatherMode": "navigation",
  "runtimeError": {
    "code": "TARGET_CRASHED",
    "message": "Browser tab has unexpectedly crashed."
  },
  "runWarnings": [
    "There may be stored data affecting loading performance in this location: IndexedDB. Audit this page in an incognito window to prevent those resources from affecting your scores.",
    "Browser tab has unexpectedly crashed."
  ],
  "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36",
  "environment": {
    "hostUserAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36",
    "benchmarkIndex": 3398.5,
    "credits": {}
  },
  "audits": {
    "is-on-https": {
      "id": "is-on-https",
      "title": "Uses HTTPS",
      "description": "All sites should be protected with HTTPS, even ones that don't handle sensitive data. This includes avoiding [mixed content](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content), where some resources are loaded over HTTP despite the initial request being served over HTTPS. HTTPS prevents intruders from tampering with or passively listening in on the communications between your app and your users, and is a prerequisite for HTTP/2 and many new web platform APIs. [Learn more about HTTPS](https://developer.chrome.com/docs/lighthouse/pwa/is-on-https/).",
      "score": null,
      "scoreDisplayMode": "error",
      "errorMessage": "Browser tab has unexpectedly crashed.",
      "errorStack": "LighthouseError: TARGET_CRASHED\n    at s1.<anonymous> (devtools://devtools/bundled/devtools-frontend/front_end/third_party/lighthouse/lighthouse-dt-bundle.js:2088:671)\n    at emitOne (devtools://devtools/bundled/devtools-frontend/front_end/third_party/lighthouse/lighthouse-dt-bundle.js:38:1045)\n    at s1.emit (devtools://devtools/bundled/devtools-frontend/front_end/third_party/lighthouse/lighthouse-dt-bundle.js:41:308)\n    at s1._handleProtocolEvent (devtools://devtools/bundled/devtools-frontend/front_end/third_party/lighthouse/lighthouse-dt-bundle.js:2088:773)\n    at devtools://devtools/bundled/devtools-frontend/front_end/third_party/puppeteer/puppeteer.js:12:332\n    at Array.map (<anonymous>)\n    at Object.emit (devtools://devtools/bundled/devtools-frontend/front_end/third_party/puppeteer/puppeteer.js:12:315)\n    at er.emit (devtools://devtools/bundled/devtools-frontend/front_end/third_party/puppeteer/puppeteer.js:12:679)\n    at er._onMessage (devtools://devtools/bundled/devtools-frontend/front_end/third_party/puppeteer/puppeteer.js:90:1813)\n    at s.onMessage (devtools://devtools/bundled/devtools-frontend/front_end/third_party/puppeteer/puppeteer.js:131:1520)"
    }
  },
  "configSettings": {
    "output": "json",
    "maxWaitForFcp": 30000,
    "maxWaitForLoad": 45000,
    "pauseAfterFcpMs": 1000,
    "pauseAfterLoadMs": 1000,
    "networkQuietThresholdMs": 1000,
    "cpuQuietThresholdMs": 1000,
    "formFactor": "mobile",
    "throttling": {
      "rttMs": 150,
      "throughputKbps": 1638.4,
      "requestLatencyMs": 562.5,
      "downloadThroughputKbps": 1474.5600000000002,
      "uploadThroughputKbps": 675,
      "cpuSlowdownMultiplier": 4
    },
    "throttlingMethod": "simulate",
    "screenEmulation": {
      "mobile": true,
      "width": 412,
      "height": 823,
      "deviceScaleFactor": 1.75,
      "disabled": true
    },
    "emulatedUserAgent": "Mozilla/5.0 (Linux; Android 11; moto g power (2022)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Mobile Safari/537.36",
    "auditMode": false,
    "gatherMode": false,
    "clearStorageTypes": [
      "file_systems",
      "shader_cache",
      "service_workers",
      "cache_storage"
    ],
    "disableStorageReset": false,
    "debugNavigation": false,
    "channel": "devtools",
    "usePassiveGathering": false,
    "disableFullPageScreenshot": false,
    "skipAboutBlank": false,
    "blankPage": "about:blank",
    "ignoreStatusCode": true,
    "locale": "en-US",
    "blockedUrlPatterns": null,
    "additionalTraceCategories": "",
    "extraHeaders": null,
    "precomputedLanternData": null,
    "onlyAudits": null,
    "onlyCategories": [
      "performance",
      "best-practices",
      "seo"
    ],
    "skipAudits": null
  }
}
```

**Comment by tlansbergen-itrs** [comment]
@adamraine  I noticed this initialy in Beta indeed, I can however reproduce in 129 Early Stable, released last Wednesday (which makes sense as it is the same buildnr. as the current beta, 129.0.6668.42). I also tested with Canary (130.0.6715.0) just now and with that build I do not see the issue.

129 Stable will be released broader starting the 17th, as it currently stands that would then include this issue.
https://chromiumdash.appspot.com/schedule
